client/src/app/pages/archetypes/components/archetype-detail-drawer.tsx

```diff
@@ -33,6 +34,24 @@ const ArchetypeDetailDrawer: React.FC<IArchetypeDetailDrawerProps> = ({
 }) => {
   const { t } = useTranslation();
 
+  const { tagCategories } = useFetchTagCategories();
+  const tags = useMemo(
+    () => tagCategories.flatMap((tc) => tc.tags).filter(Boolean),
+    [tagCategories]
+  );
+
+  const archetypeTags =
+    archetype?.tags
+      ?.filter((t) => t?.source ?? "" === "")
+      .map((ref) => tags.find((tag) => ref.id === tag.id))
+      .filter(Boolean) ?? [];
+
+  const assessmentTags =
+    archetype?.tags
+      ?.filter((t) => t?.source ?? "" !== "")
```

Comment:
Are we looking to filter by assessment source here?

Reply:
Like the filters in the tags tab of the application drawer?

Reply:
![image](https://github.com/konveyor/tackle2-ui/assets/3985964/d1ec8ca3-61dc-43b7-b592-8c3162ba376d)

Reply:
Ok. It just wasn't clear to me exactly what the empty source was meaning in this context.

Reply:
I was looking for the actual "assessment" source type like we see here 
<img width="372" alt="Screenshot 2023-10-05 at 5 09 19 PM" src="https://github.com/konveyor/tackle2-ui/assets/11218376/7eaa4c7f-c8ae-4609-9fc9-14285924e3e2">
<img width="1880" alt="Screenshot 2023-10-05 at 5 10 48 PM" src="https://github.com/konveyor/tackle2-ui/assets/11218376/dffe677f-fa7e-47ec-9c5a-0ba402ad0645">

With this logic, it looks like the assessment tags are going to be categorized as archetype tags && assessment tags.

Reply:
Yeah... once I can run an assessment on an archetype I'll be able to verify that's what it would look like.

Reply:
The logic is...
  - if the tag source is undefined or empty, it is an archetype tag
  - if the tag source is not empty, it is an assessment tag